icalingua/src/renderer/components/vac-mod/ChatWindow/Room/Room.vue

```diff
@@ -187,13 +187,41 @@
                     </div>
                 </div>
 
-                <transition name="vac-fade-quickface">
-                    <QuickFace
-                        v-show="isQuickFaceOn"
+                <transition name="vac-fade-search-input">
+                    <SearchInput
                         ref="quickface"
+                        v-show="isQuickFaceOn"
+                        v-slot="{ id, name }"
+                        :list="faceNames"
+                        :description="'face(s)'"
+                        :searchMethod="'startsWith'"
+                        :inputSize="80"
                         @cancel="closeQuickFace"
                         @confirm="useQuickFace"
-                    />
+                    >
+                        <p>{{ name }}</p>
+                        <img :src="`file://${faceDir}/${id}`" />
+                    </SearchInput>
+                </transition>
+
+                <transition name="vac-fade-search-input">
+                    <SearchInput
+                        ref="quickat"
+                        v-show="isQuickAtOn && room.roomId < 0"
+                        v-slot="{ id, name }"
+                        :list="groupMembers
+                            ? groupMembers.map(({ card, nickname, user_id }) => [ card || nickname, user_id ])
+                            : []
+                        "
+                        :description="'member(s)'"
```

Comment:
应该不会出现一个 member 的情况吧？
而且这里似乎用不上绑定值
```suggestion
                        description="members"
```

Reply:
> 应该不会出现一个 member 的情况吧？ 而且这里似乎用不上绑定值

- 数量是根据输入筛选实时更新的，会出现 0、1 的情况
- 确实不用绑定值，我改一下

Reply:
似乎我记得 0 应该也是复数（？）

Reply:
> 似乎我记得 0 应该也是复数（？）

Confirmed.